server/models/userCreate.js

```diff
@@ -0,0 +1,56 @@
+import { hashSync } from 'bcryptjs';
+
+import userData, { allUsers } from './userData';
+
+class User {
+    constructor() {
+        this.message;
+        this.findId = (id) => {
+            let foundId = allUsers.find(userId => userId.id === id);
+            return foundId;
+        };
+        this.createUser = (newUser) => {
+            // if (typeof(newUser) == "object")
+            const usedEmail = userData.allUsers.find(userEmail => userEmail.email === newUser.email);
+            if (usedEmail !== 'undefined') {
+                let inputData = {
+                    id: allUsers.length + 1,
+                    email: newUser.email || '',
```

Comment:
If any of the user properties are missing(falsy), you should simply not create the user. Do not default to an empty string.